server/src/test/java/io/crate/analyze/AlterTableAddColumnAnalyzerTest.java

```diff
@@ -278,6 +280,39 @@ public void testAddNewNestedObjectColumn() throws Exception {
         assertThat((String) y.get("type"), is("keyword"));
     }
 
+    @Test
+    public void test_check_constraint_can_be_added_to_nested_object_() throws Exception {
+        e = SQLExecutor.builder(clusterService)
+            .addTable("""
+                create table t (
+                    i int,
+                    o object
+                )
+            """)
```

Comment:
probably more my personal preference, a sin.le line has the same readability for me for such a simple stmt,
```suggestion
            .addTable("create table t (i int, o object)")
```